tests/emulators/default_q0/model.json

```diff
@@ -0,0 +1,36 @@
+{
+    "device_name": "ibmfakebelem_q0",
+    "topology": [],
+    "nqubits": 1,
+    "ncouplers": 0,
+    "qubits_list": [
+        "0"
+    ],
+    "couplers_list": [],
```

Comment:
I am pretty sure this has been discussed before (sorry for repeating), but what is the motivation of duplicating some parameters in `model.json` and `parameters.json`? The particular ones highlighted here are more or less constants so I don't think we need them twice. Also, T1, T2 and frequencies exist in `parameters.json` too but with different values.

In case of duplication, I would prefer keeping these only in `parameters.json`, which also exist for hardware platforms and thus will make the transition between the two easier.

Reply:
My motivation was to have an emulator whose model is determined completely by `model.json` and independent of the runcard, i.e. `parameters.json`, which would correspond to what is in the lab or obtained from calibration. In the ideal situation when one has the exact model describing the device, indeed the parameters in both files should be the same. But this is not the case in practice, and therefore there is a possibility that one has to readjust some of these parameters for the emulator in order to reproduce something close to experiments. @sorewachigauyo and I have already encountered this issue, in which the rabi frequency has to be artificially boosted by 10 in simulations in order to get something reasonable. This could be the fault of the model, or some units conversion or something else, and having a decoupled set of parameters for the emulator would allow us to check such issues. Other than this known issue, the different values between `parameters.json` and `model.json` is just because I did not update those in `parameters.json` since I did not use them, but yes I will update them if we choose to keep this arrangement.

Basically my idea with this arrangement is for `parameters.json` be an exact copy of that from the device, so that a model (both the model and the model's input parameters) can be tweaked to match the outputs of the device based on the same pulse settings from `parameters.json`, even if it means adjusting some of the parameters to be different from what is reported from the device. The main benefit is that it would be clear which parameters from the runcard are currently used by the model, and where there are discrepancies, the values can also be directly compared. 

If we don't have duplicate parameters in `model.json`, then it is up to the onus of the emulator user to keep track of which parameters are used by the model (a lot of the information in `parameters.json` are not), and which values are artificially adjusted from the device. In this case, then instead of `model.json` only including additional parameter required by the emulator model that are currently not in the runcard e.g. coupling strength, it would be better to only have `parameters.json` and enlarge the dictionary to include all the additional parameters that would otherwise be stored in `model.json`, which @sorewachigauyo suggested. This should be ok for the time being. In this new arrangement, one has to be careful when updating `parameters.json` of the emulator from the device, while in the current arrangement, one can do so freely and it is clear from the setup that only the pulses are updated in the emulator while the model is not since it is based on `model.json`.

Reply:
I'd rather suggest having to fully distinct platforms, each with its own `parameters.json`:
```
lab-x-chip-y/
  platform.py
  parameters.json
  ...
lab-x-chip-y-emulator/
  platform.py
  parameters.json
  ...
```
also because even the `platform.py` will be quite different (if not completely).

If you'll be able to map the `parameters.json` of the target physical platform to those of the emulator, you can write a function doing that.

Reply:
> I'd rather suggest having to fully distinct platforms, each with its own `parameters.json`:
> 
> ```
> lab-x-chip-y/
>   platform.py
>   parameters.json
>   ...
> lab-x-chip-y-emulator/
>   platform.py
>   parameters.json
>   ...
> ```
> 
> also because even the `platform.py` will be quite different (if not completely).

Sure, we can work with this setup.
 
> If you'll be able to map the `parameters.json` of the target physical platform to those of the emulator, you can write a function doing that.

This might be messy and largely hypothetical at this stage, will implement some version of this after the emulator has seen some use to get a good sense of the most relevant version of this function.

> If possible, I'd drop entirely `model.json`.

Will do.

Reply:
Integrated model.json and simulation_config into parameters.json, and removed model.json in 4e6cc09b9198fb0e218e0dd08a5f539dad122e19